databricks/sdk/core.py

```diff
@@ -892,6 +892,30 @@ def copy(self):
         return cpy
 
 
+# Known ErrorDetail types
+errorInfoType = "type.googleapis.com/google.rpc.ErrorInfo"
+
```

Comment:
This should move into the DatabricksError class and be prefixed with a `_` to indicate it's not a part of the public API.

Also snake-case please.